docs/kratos/self-service/flows/verify-email-account-activation.mdx

```diff
@@ -183,408 +110,129 @@ your Identity Schema:
-  },
-  "state": "choose_method"
-}
-```
+For more information see the [hooks configuration](../../hooks/01_configure-hooks.mdx) documentation.
 
-### Send verification link to email
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
 
 :::note
 
-The `link` verification mode will always open a link in the browser, even if it was initiated by an API client. This is because
-the user clicks the link in his/her email client, which opens the browser.
+Please be aware, that when requiring a verified address for login, a typo in the email address set by the user either during the
+registration or when changing their email address, will make the login for that user impossible.
 
 :::
 
-When the `link` method is enabled, it will be part of the `methods` payload in the Verification Flow:
-
-```shell script
-curl -H "Accept: application/json" -s \
-    'https://playground.projects.oryapis.com/self-service/verification/flows?id=956c0499-17a9-4a99-9602-9282d1632f7b' | \
-      jq -r '.ui.nodes[] | select(.group=="link")'
-
-{
-  "type": "input",
-  "group": "link",
-  "attributes": {
-    "name": "email",
-    "type": "email",
-    "required": true,
-    "disabled": false
-  },
-  "messages": null,
-  "meta": {}
-}
-{
-  "type": "input",
-  "group": "link",
-  "attributes": {
-    "name": "method",
-    "type": "submit",
-    "value": "link",
-    "disabled": false
-  },
-  "messages": null,
-  "meta": {
-    "label": {
-      "id": 1070005,
-      "text": "Submit",
-      "type": "info"
-    }
-  }
-}
-```
+## Choosing the right strategy
 
-## Verification form validation
+Ory supports two strategies for verifying your user's addresses.
 
-The form payloads are then submitted to Ory Identities which follows up with:
+```mdx-code-block
+<Tabs>
+  <TabItem value="code" label="One-time codes">
+```
 
-- An HTTP 303 See Other redirect pointing to the Registration UI for Browser Clients;
-- An `application/json` response for API Clients and Client-Side Browser applications (for example Single Page Apps)e Apps).
+When using one-time codes to verify a user's address, Ory Identities sends out an email containing a 6-digit code. If the user has
+access to the mailbox they can enter the code they received to verify their address.
 
-### Verification Link via Email
+Alternatively, the email also contains a link to the form that pre-fills the code if the user clicks on it. The user must still
+submit the form for Ory Identities to actually mark the address as verified.
 
-To send the verification email, the end-user fills out the form. There might be validation errors such as a malformed email:
+```mdx-code-block
+  </TabItem>
+  <TabItem value="oss" label="Magic Links">
+```
 
-<CodeTabs items={getFlowMethodLinkWithErrors} />
+When choosing to send magic links to verify a user's address, Ory Identities sends out an email containing a "magic link". If the
+user has access to the mailbox of that account and clicks the link, Ory Identities marks that address as verified.
```

Comment:
```suggestion
When using the "magic link" strategy, Ory Identities sends the user an email with a "magic link". If the
user has access to the email associated with the and clicks the link, Ory Identities marks that address as verified.
```

Reply:
I used "strategy", but shouldn't it be "method" as mentioned earlier?

Reply:
Done.